python/setup.py

```diff
@@ -84,6 +84,10 @@ def get_cuda_version_from_header(cuda_include_dir):
 except Exception:
     nthreads = 0
 
+define_macros = [
+    ("CUDA_API_PER_THREAD_DEFAULT_STREAM", None),
```

Comment:
So does this make this a command line option? How does this work? (Not that familiar with setup.py)